src/config/ActionTypes.js

```diff
@@ -0,0 +1,11 @@
+const REGISTER_REQUEST = 'REGISTER_REQUEST';
```

Comment:
Try doing `export const REGISTER_REQUEST = 'REGISTER_REQUEST'`. You won't have to write the constant names again.

Reply:
Done!